pythonforandroid/toolchain.py

```diff
@@ -739,6 +739,15 @@ def _read_configuration():
                 sys.argv.append(arg)
 
     def recipes(self, args):
+        """
+        Prints recipes basic info, e.g.
+        ```
+        python3      3.7.1
+            depends: ['hostpython3', 'sqlite3', 'openssl', 'libffi']
+            conflicts: ['python2']
+            optional depends: ['sqlite3', 'libffi', 'openssl']
+        ```
+        """
```

Comment:
**minor:** I'm not sure how it will be rendered this docstring in sphinx...maybe I would use a `code-block` directive, something like:

```
        Prints recipes basic info, e.g.
        .. code-block:: bash

            python3      3.7.1
            depends: ['hostpython3', 'sqlite3', 'openssl', 'libffi']
            conflicts: ['python2']
            optional depends: ['sqlite3', 'libffi', 'openssl']
 ```

But, @AndreMiras, don't need to change this, if I remember well, I think that we don't have enabled the build of technical documentation, so it wont be visible for now.

Apart from this, I would like to mention that overall it looks good, clean and very readable.Also, I made tox test on my machine, with current develop branch , since it's already merged, without any issue.

@AndreMiras, thanks for this PR, good job, as usual :+1:

and @JonasT, thanks for your review, I was out for the weekend and I was unable to take care of the review, so yours sped up the merging :smile:

Reply:
I don't know how I missed that comment. I'd be happy to address it still